### ISSUE TITLE
onChange prop of redux form passed through

### ISSUE THREAD

**Issue by tkvw** [erikras/redux-form#3772]
The onChange prop of `reduxForm` is passed through, this is not documented, is this by design?
 
This example shows the behaviour: 
https://codesandbox.io/s/n9q85r9wp0

**Comment by gustavohenke** [comment]
>  is this by design?
I don't think so.
If you don't use it within the form it'll still work: https://codesandbox.io/s/1v1ylmm763

Would you like to remove it?

**Comment by tkvw** [comment]
Yes, I think the `reduxForm` should only add props that are documented. I don't see the need of passing on the `onChange` prop.  In my case it added the `onChange` prop to all `Field`  components, because I am passing on all props to the child components.

**Comment by aditya9026** [comment]
`def raise_capital_saved
		if params[:_detail] && params[:_detail] == "transaction" && session.delete(:has_raised_capital)
			@message_for = "transaction"
			UserMailer.transaction_intake_submittted(current_user, "New transaction intake").deliver
		elsif params[:_detail] && params[:_detail] == "sponsor" && session.delete(:has_raised_capital)
			@message_for = "sponsor"
		else
			redirect_to raise_capital_path
		end
	end`

**Comment by aditya9026** [comment]
```def raise_capital_saved`
``		if params[:_detail] && params[:_detail] == "transaction" && session.delete(:has_raised_capital)`
`			@message_for = "transaction"`
`			UserMailer.transaction_intake_submittted(current_user, "New transaction intake").deliver`
``		elsif params[:_detail] && params[:_detail] == "sponsor" && session.delete(:has_raised_capital)``
			`@message_for = "sponsor"`
``		else``
			`redirect_to raise_capital_path`
		end``
end```

**Comment by aditya9026** [comment]
`def test
`` put x``
end`

**Comment by anu1097** [comment]
I am sorry to jump on this. But I don't understand the issue clearly. OnChange is one prop which I have heavliy used in my projects, what is it exactly you want to enhance ? I would love to work on this.
